### PR TITLE
change .env

### PR DESCRIPTION
**Pull request by Miminiverse**
update .ENV

### CODE REVIEW

**Review by Said-laasri — APPROVED** [review]
@Miminiverse  Great Job :+1: 



HAPPY CODING!!!

File: index.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="/favicon.ico" />
+    <link rel="icon" href="./src/assests/cockatoo.png" />
```
Comment (royemosby) [inline index.html:5]: Be vary careful with this. That src directory does not get transferred over into the build directory that gets created during the build/deploy process. More information can be found in the Vite docs [here](https://vitejs.dev/guide/assets.html) and [here](https://vitejs.dev/guide/features.html#static-assets). Please confirm that you have a working URL here then ping me on Slack.

**Review by royemosby — CHANGES_REQUESTED** [review]
Please confirm asset url

File: index.html (hunk 1 above)
Comment (frankstepanski) [reply index.html:5]: Excellent point. Definitely good practice for what changes need to be made to a code-base when you deploy it.

Comment (Miminiverse) [reply index.html]: Got it! I moved the image to public folder and use the absolute path.

Comment (royemosby) [reply index.html]: 👍 schweet!

**Review by royemosby — APPROVED** [review]
GTG!

**Pull request merged**